platforms/android/app/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.maq.xprize.cci.hindi"
     android:hardwareAccelerated="true"
-    android:versionCode="1"
+    android:versionCode="2"
     android:versionName="1.0.0">
```

Comment:
Make this 2.0.0. This should be as per the release version in GitHub releases

Reply:
Updated made it 3.0.0